targets/csharp/source/PlayFabSDK/source/PlayFabHttp/PlayFabHttp.cs

```diff
@@ -35,6 +35,7 @@ public class PlayFabJsonError
         public int errorCode;
         public string errorMessage;
         public Dictionary<string, string[]> errorDetails = null;
+        public uint? retryAfterSeconds = null;
```

Comment:
An issue popped up on line 409 of this file in our Unit tests from the nullable uint here:

"Assets\PlayFabSDK\Shared\Internal\PlayFabHttp\PlayFabHTTP.cs(409,37): error CS0173: Type of conditional expression cannot be determined because there is no implicit conversion between 'uint' and '<null>'"